docs/js_library.md

```diff
@@ -41,8 +41,8 @@ Can be used as a dep for rules that expect a DeclarationInfo such as ts_project.
 | Name  | Description | Type | Mandatory | Default |
 | :------------- | :------------- | :------------- | :------------- | :------------- |
 | <a id="js_library-name"></a>name |  A unique name for this target.   | <a href="https://bazel.build/docs/build-ref.html#name">Name</a> | required |  |
-| <a id="js_library-deps"></a>deps |  -   | <a href="https://bazel.build/docs/build-ref.html#labels">List of labels</a> | optional | [] |
-| <a id="js_library-srcs"></a>srcs |  -   | <a href="https://bazel.build/docs/build-ref.html#labels">List of labels</a> | optional | [] |
+| <a id="js_library-deps"></a>deps |  Dependencies of this library.<br><br>        This may include other js_library targets or targets that provider JsInfo as well as         linked npm dependencies.   | <a href="https://bazel.build/docs/build-ref.html#labels">List of labels</a> | optional | [] |
+| <a id="js_library-srcs"></a>srcs |  The list of source files that are included in this library.<br><br>        This includes all your checked-in code and any generated source files.<br><br>        Other js_library targets and npm dependencies belong in <code>deps</code>.   | <a href="https://bazel.build/docs/build-ref.html#labels">List of labels</a> | optional | [] |
```

Comment:
Does "checked-in" along with "any generated source files" exclude anything? Could we just say "source files"?

idk if mentioning "Other js_library targets" is really necessary either, especially since it was mentioned on the previous line 🤷

Reply:
> Thank you for doing many isolated commits 👍
> 
> Should we try to merge some of those sooner to avoid merge conflicts in this PR?

I've got some more work to do before any of it is ready. Will probably land as one big stack since it will be hard to reorder and squash this one.

Reply:
I borrowed the working from py_library srcs: https://docs.bazel.build/versions/main/be/python.html#py_library

Remove the "Other js_library targets" bit from the docstring.